app/models/something.rb

```diff
@@ -0,0 +1,9 @@
+class Something < ActiveRecord::Base
```

Comment:
[IrresponsibleModule](https://github.com/troessner/reek/blob/master/docs/Irresponsible-Module.md): has no descriptive comment
Lines: [1]

[UncommunicativeMethodName](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Method-Name.md): has the name 'a'
Lines: [2]

[UncommunicativeMethodName](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Method-Name.md): has the name 'b'
Lines: [6]

[UtilityFunction](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md): doesn't depend on instance state (maybe move it to another class?)
Lines: [2]